docs/access/client.md

```diff
@@ -0,0 +1,240 @@
+
+### Client profile setting
+
+With `CVMFS_CLIENT_PROFILE="single"` we specify that this CernVM-FS client should:
+
+* Use the [proxy server](../appendix/terminology.md#proxy) specified via `CVMFS_HTTP_PROXY`, if that configuration setting is defined;
+* Directly connect to a [Stratum-1 replica server](../appendix/terminology.md#stratum1) that provides the repository being used if no proxy server
+  is specified via `CVMFS_HTTP_PROXY`.
+
+As an alternative to defining `CVMFS_CLIENT_PROFILE`, you can also set `CVMFS_HTTP_PROXY` to `DIRECT` to specify
+that no proxy server should be used by CernVM-FS:
+
+``` { .ini .copy }
+CVMFS_HTTP_PROXY="DIRECT"
+```
+
+We will get back to `CVMFS_HTTP_PROXY` later when [setting up a proxy server](proxy.md).
+
+### Quota limit for client cache
+
+The `CVMFS_QUOTA_LIMIT` configuration setting specifies the maximum size of the CernVM-FS client cache (in MBs).
+
+In the example above, we specify that no more than ~10GB should be used for the client cache.
+
+When the specified quota limit is reached, CernVM-FS will automatically remove files from the cache according
+to the [Least Recently Used (LRU) policy](https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU),
+until half of the maximum cache size has been freed.
+
+The location of the cache directory can be controlled by `CVMFS_CACHE_BASE` if needed (default: `/var/run/cvmfs`),
+but **must** be a on a *local* file system of the client, not a network file system that can be modified by multiple
+hosts.
+
+Using a directory in a RAM disk (like `/dev/shm`) for the CernVM-FS client cache
+is possible but not recommended in general, mostly because the internal implementation
+in CernVM-FS to support this is pretty involved, and not extensively tested.
+
+For more information on cache-related configuration settings,
+[see the CernVM-FS documentation](https://cvmfs.readthedocs.io/en/stable/cpt-configure.html#cache-settings).
+
+
+## Completing the client setup
+
+To complete the setup of the CernVM-FS client component,
+a`cvmfs` user account and group must be added to the system,
+and the `/cvmfs` and `/var/lib/cvmfs` directories must be created.
+
+In addition, it is recommended to update the `autofs` configuration
+to enable auto-mounting of CernVM-FS repositories *(optional)*.
+
+All these actions can be performed in one go by running the following command:
+
+``` { .bash .copy }
+sudo cvmfs_config setup
+```
+
+Additional options can be passed to the `cvmfs_config setup` command to disable some of the actions,
+like `nouser` to not create the `cvmfs` user and group, or `noautofs` not
+update the `autofs` configuration.
+
+### Recommendations for `autofs`
+
+It is recommended to increase the timeout used by `autofs` for unmounting due to inactivity,
+by setting additional options in `/etc/sysconfig/autofs`, for example:
+
+```{ .ini .copy }
+OPTIONS="--timeout 3600"
+```
+
+This sets the timeout for `autofs` to 1 hour (3600 seconds).
```

Comment:
I think the better recommendation would be `-t 0` or statically mount CVMFS repos, whenever you are running a RHEL7 or RHEL 8 distribution.
This shouldn't be an issue for RHEL 9 distribution.

This is to circumvent an issue where containers are not interacting nicely (see https://github.com/cvmfs/cvmfs/issues/3402#issuecomment-1761370983 )
Ideally you'd fix the workflow in the containers, but if you are running a cluster environment, you can't in many cases.
We tried `-t 3600` as a "ridiculously large timeout", which still wasn't large enough in some cases.